webui/src/pages/repositories/repository/objects.jsx

```diff
@@ -182,8 +182,8 @@ const ObjectsBrowser = ({ config, configError }) => {
                         withWorkspace={true}
                         selectRef={ref => router.push({
                             pathname: `/repositories/:repoId/objects`,
-                            params: {repoId: repo.id},
-                            query: {ref: ref.id}
+                            params: {repoId: repo.id, path:path},
+                            query: {ref: ref.id, path: path}
```

Comment:
I get `undefined` as path if I'm switching between branches without any path:
![image](https://user-images.githubusercontent.com/9861474/136160253-99f995d8-d6cd-4f9f-8e88-26b69df3b231.png)

Reply:
Oops, fixed it, thanks!